### ISSUE TITLE
Inelegance in error reporting

### ISSUE THREAD

**Issue by jelavich** [codex-protocol/web.codex-viewer#51]
When I went to go from a record window to the Collections tab, I got this message, which was correct because I wasn't logged in: "Could not get collection: The specified auth token is invalid." But after I logged in, the message didn't vanish regardless of what tab I selected, or what record I opened. Minor, but inelegant.

**Comment by Anaphase** [comment]
Error messages are intentionally pinned until you dismiss them, unlike success messages which disappear after 5 seconds. The intent was to make it so the user would never accidentally miss an error message by looking away from the screen for 5 seconds.

@johnhforrest / @sprice what do you think? SHould error messages be dismissed after some timeout?

**Comment by jelavich** [comment]
I understand that need of making it clear to people those error messages, but once I've cleared the fault (not logged in), I think that the message, now no longer valid, (flag unset), should go. Otherwise I'm getting a message that ins't true, and I (at least personally) would get set down the wrong path, hunting for what is wrong, even though everything is good now. Error messages are off-putting, and for non-tech people can create panic. The tech is supposed to work FOR US, not become a yoke or anchor.

**Comment by johnhforrest** [comment]
Makes sense to clear this one upon login. I also think this error message can be a little bit more user friendly. e.g., "Your login has timed out" instead of talking about auth tokens.

**Comment by jelavich** [comment]
Agreed on the more friendly error msg.
